src/plays/model/PlayCreateRequest.ts

```diff
@@ -0,0 +1,6 @@
+import PlayScheduleConfiguration from "./PlayScheduleConfiguration";
```

Comment:
@appu-channelape Change this to the single quotes like the build server is asking